scripts/lib/install_cassandra.sh

```diff
@@ -0,0 +1,35 @@
+#
+# Cassandra Installation Script
+# Reference: http://cassandra.apache.org/download/
+#
+# Install Java 8 if not installed
+install_java_environment() {
+  command -v java >/dev/null 2>&1;
+  if [[ $? -eq 1 ]]; then
+    echo "Java not installed..."
+    echo "Installing java runtime..."
+    sudo apt-get -y install openjdk-8-jre
```

Comment:
I believe that if you install the jdk it already has the jre in it so you won't need this line.